src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1393,19 +1393,20 @@ public String getFeedbackSessionResultSummaryAsCsv(
      * Preconditions: <br>
      * * All parameters(except questionId) are non-null. <br>
      * @see FeedbackSessionsLogic#getFeedbackSessionResultsSummaryInSectionAsCsv(String, String, String,
-     *      String, String, boolean, boolean)
+     *      String, String, String, boolean, boolean)
      */
     public String getFeedbackSessionResultSummaryInSectionAsCsv(
             String courseId, String feedbackSessionName, String instructorEmail,
-            String section, String questionId, boolean isMissingResponsesShown, boolean isStatsShown)
+            String section, String sectionDetail, String questionId, boolean isMissingResponsesShown, boolean isStatsShown)
```

Comment:
`sectionDetails` should be type of the enum you defined right?

Reply:
I used string instead as the `FeedbackSessionsLogic.java` has multiple usage of sectionDetail in terms of String as it is put into a `Map<String, String> params` so I cannot put in an enum instead. Hence, I decided to change it to String for `Logic` onwards instead of changing all the existing implementations.

Reply:
it is put into a Map<String, String> params so I cannot put in an enum instead.

Using String will loss the point of using enum. Let's see if we can find any workaround.

Reply:
I can pass in a separate `SectionDetail` parameter to the functions if the selectedSectionDetail is not `null`?